README.md

```diff
@@ -62,48 +62,39 @@ Two imporant parameters that one needs to verify in the central nanoAOD document
 @BTV-Commissioning-Team: the recommended PFNano customization for data is `PFnano_customizeData_add_DeepJet` and for MC `PFnano_customizeMC_add_DeepJet_and_Truth`.
 
 <details>
-    <summary>Here are five example commands, with which the runnable configs in `test` have been created:</summary>
+    <summary>Here are four example commands, with which the runnable configs in `test` have been created:</summary>
     
     
 ```
-cmsDriver.py nano_data_2022ABCD --data --eventcontent NANOAOD --datatier NANOAOD --step NANO \
---conditions 124X_dataRun3_v11   --era Run3,run3_nanoAOD_124 \
---customise_commands="process.add_(cms.Service('InitRootHandlers', EnableIMT = cms.untracked.bool(False)));process.MessageLogger.cerr.FwkReport.reportEvery=1000;process.NANOAODoutput.fakeNameForCrab = cms.untracked.bool(True)" --nThreads 4 \
--n -1 --filein "/store/data/Run2022C/DoubleMuon/MINIAOD/10Dec2022-v1/2820000/dea1757f-d2ef-467a-9062-714775d00e45.root" --fileout file:nano_data2022ABCD.root \
---customise="PhysicsTools/PFNano/pfnano_cff.PFnano_customizeData_add_DeepJet"  --no_exec
-```
-<br>
-```
-cmsDriver.py nano_data_2022E --data --eventcontent NANOAOD --datatier NANOAOD --step NANO \
---conditions 124X_dataRun3_v14   --era Run3,run3_nanoAOD_124 \
+cmsDriver.py nano_data_2022CDE --data --eventcontent NANOAOD --datatier NANOAOD --step NANO \
+--conditions 124X_dataRun3_v15   --era Run3,run3_nanoAOD_124 \
```

Comment:
Are those really the GTs for 2022 Partial ReReco Data? I find some contradicting information: [nanoAOD v11 wiki](https://gitlab.cern.ch/cms-nanoAOD/nanoaod-doc/-/wikis/Releases/NanoAODv11) vs [PdmV](https://twiki.cern.ch/twiki/bin/viewauth/CMS/PdmVRun3Analysis#Early_Analyses_Information_AN1), maybe we should double check with PPD?

Reply:
Good catch! I think the PdmV link is the latest one. From https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/124X_dataRun3_PromptAnalysis_v2 
it indicates "PromptAnalysis GT copied from 124X_dataRun3_Prompt_v10 with updated JEC/JR for 2022FG".

Reply:
Hm, yeah but you are referring to the Prompt GT, my question is about the ReReco GT: the first link I mentioned lists v14, while PdmV lists v15, and I am not sure if v15 is already the GT for the partial ReReco or if v15 will be the GT to be used for ReReco v2?

Reply:
ah, yes! By partial ReReco do you main the previous data with [10Dec2022] tag, and ReReco v2 refers to [27Jun2023]? If this is the case, we should already move to 27Jun2023 (both analysis, our SF derivation etc)

Reply:
Ok, I might have confused myself concerning the meaning of Partial ReReco: Partial ReReco seems to be the one with the June2023 tag, see slide 7 of [PPD news](https://indico.cern.ch/event/1312617/contributions/5521520/attachments/2694090/4675543/23-08-03_News_PPD.pdf) and the corresponding [pMp link](https://cms-pdmv-prod.web.cern.ch/pmp/historical?r=27Jun2023&showDoneRequestsList=true). So I conclude that the GTs in your PR are the correct ones

Reply:
ah I see. Thanks!